app/src/main/java/com/sakal/playlistmaker/media_library/ui/playlists/PlaylistsFragment.kt

```diff
@@ -0,0 +1,103 @@
+package com.sakal.playlistmaker.media_library.ui.playlists
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.coroutineScope
+import androidx.navigation.fragment.findNavController
+import com.sakal.playlistmaker.R
+import com.sakal.playlistmaker.databinding.FragmentPlaylistsBinding
+import com.sakal.playlistmaker.media_library.ui.adapters.PlaylistsAdapter
+import com.sakal.playlistmaker.media_library.ui.viewmodels.PlaylistsScreenState
+import com.sakal.playlistmaker.media_library.ui.viewmodels.PlaylistsViewModel
+import com.sakal.playlistmaker.new_playlist.domain.models.Playlist
+import kotlinx.coroutines.launch
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class PlaylistsFragment : Fragment() {
+
+    private lateinit var binding: FragmentPlaylistsBinding
+    private val viewModel by viewModel<PlaylistsViewModel>()
+    private val playlistsAdapter = PlaylistsAdapter {
+        clickOnPlaylist()
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ): View {
+        binding = FragmentPlaylistsBinding.inflate(inflater, container, false)
+        return binding.root
+
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        initAdapter()
+
+        viewLifecycleOwner.lifecycle.coroutineScope.launch {
```

Comment:
```suggestion
        viewLifecycleOwner.lifecycleScope.launch {
```
Можно так написать, чтоб было короче